Commons/src/main/java/eu/neclab/ngsildbroker/commons/constants/NGSIConstants.java

```diff
@@ -179,4 +179,6 @@
 	public static final String MQTT_QOS = "mqtt_qos";
 	public static final String MQTT_VERSION = "mqtt_version";
 	public static final String DEFAULT_DATA_SET_ID = "https://uri.etsi.org/ngsi-ld/default-data-set-id";
```

Comment:
DEFAULT_DATA_SET_ID_VALUE and DEFAULT_DATA_SET_ID seem to have the use and only one is used. 
If that's the case please remove the one you are not using

Reply:
Both are using, one is using dataset Id uri and other one dataset Id default value.

Reply:
Nope the dataset uri is 
`public static final String NGSI_LD_DATA_SET_ID = "https://uri.etsi.org/ngsi-ld/datasetId";`
DEFAULT_DATA_SET_ID id is supposed to be the default value if none other is given that's why it's set if none is given in the BaseEntry. DEFAULT_DATA_SET_ID_VALUE is now doing the same in another spot. DEFAULT_DATA_SET_ID_VALUE  should be replaced by DEFAULT_DATA_SET_ID.

Reply:
I think Benni is right.
Below is my understanding.
Below is the key for data-set-ID:
public static final String NGSI_LD_DATA_SET_ID = "https://uri.etsi.org/ngsi-ld/datasetId";
Below is the value for default data-set-Id
public static final String DEFAULT_DATA_SET_ID = "https://uri.etsi.org/ngsi-ld/default-data-set-id";
Below is redundant  for value of  default data-set-Id :
public static final String DEFAULT_DATA_SET_ID_VALUE = "######@@";

it will be perfectly fine to replace DEFAULT_DATA_SET_ID_VALUE with DEFAULT_DATA_SET_ID

Reply:
Replaced